media/js/products/vpn/invite.js

```diff
@@ -0,0 +1,177 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var newsletterForm = document.getElementById('newsletter-form');
+
+    if (!newsletterForm) {
+        return;
+    }
+
+    // handle errors
+    var errorArray = [];
+    var newsletterErrors = document.getElementById('newsletter-errors');
+    var newsletterContent = document.querySelector('.mzp-c-newsletter-content');
+
+    function disableFormFields() {
+        var formFields = newsletterForm.querySelectorAll('input, select');
+
+        for (var i = 0; i < formFields.length; i++) {
+            formFields[i].disabled = true;
+        }
+    }
+
+    function enableFormFields() {
+        var formFields = newsletterForm.querySelectorAll('input, select');
+
+        for (var i = 0; i < formFields.length; i++) {
+            formFields[i].disabled = false;
+        }
+    }
+
+    function newsletterError() {
+        enableFormFields();
+
+        if (errorArray.length) {
+
+            // create error list container if it does not exist.
+            if (!newsletterErrors) {
+                newsletterErrors = document.createElement('div');
+                newsletterErrors.id = 'newsletter-errors';
+                newsletterErrors.className = 'mzp-c-form-errors';
+                newsletterContent.insertBefore(newsletterErrors, newsletterContent.firstChild);
+            }
+
+            var errorList = document.createElement('ul');
```

Comment:
nit: Add a class name of `mzp-u-list-styled` to this `ul`. This will give it the same bulleted look as demoed on the [Protocol site](https://protocol.mozilla.org/patterns/atoms/forms.html#error-list). 

Protocol site: 
<img width="426" alt="image" src="https://user-images.githubusercontent.com/2692333/108176965-3e8b0380-70c8-11eb-830a-21cd58d63e7b.png">


moz.org/vpn: 
<img width="744" alt="image" src="https://user-images.githubusercontent.com/2692333/108177015-4cd91f80-70c8-11eb-89ef-d978480f811d.png">

Reply:
Good catch!